sdk/face/Azure.AI.Vision.Face/src/Custom/LargeFaceListClient.cs

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.Vision.Face
+{
+    // Data plane generated sub-client.
+    /// <summary> The LargeFaceList sub-client. </summary>
+    [CodeGenClient("LargeFaceListClientImpl")]
+    public partial class LargeFaceListClient
+    {
+        /// <summary> Initializes a new instance of FaceClient. </summary>
+        /// <param name="endpoint">
+        /// Supported Cognitive Services endpoints (protocol and hostname, for example:
+        /// https://{resource-name}.cognitiveservices.azure.com).
+        /// </param>
+        /// <param name="largeFaceListId"> Valid character is letter in lower case or digit or '-' or '_', maximum length is 64. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public LargeFaceListClient(string largeFaceListId, Uri endpoint, AzureKeyCredential credential) : this(largeFaceListId, endpoint, credential, new AzureAIVisionFaceClientOptions())
```

Comment:
Put `largeFaceListId` parameter behind the endpoint and credential 
```suggestion
        public LargeFaceListClient(Uri endpoint, AzureKeyCredential credential, string largeFaceListId) : this(endpoint, credential, largeFaceListId, new AzureAIVisionFaceClientOptions())
```

Reply:
Updated